src/vite-env.d.ts

```diff
@@ -1,7 +1,13 @@
 /// <reference types="vite/client" />
 
+declare module '@agoric/web-components' {
+  export const makeAgoricKeplrConnection;
```

Comment:
consider copying this over to make refactoring easier. and so you don't rely on agoric-sdk releases.

Reply:
I still need the @agoric/web-components/react component for the bridge so I'll just leave this for now